executor/simple_test.go

```diff
@@ -394,6 +394,10 @@ func (s *testSuite3) TestFlushPrivileges(c *C) {
 	// After flush.
 	_, err = se.Execute(ctx, `SELECT Password FROM mysql.User WHERE User="testflush" and Host="localhost"`)
 	c.Check(err, IsNil)
+
+	config.GetGlobalConfig().Security.SkipGrantTable = true
+	tk.MustExec("FLUSH PRIVILEGES")
```

Comment:
I run this test in the master branch, it passed. I think it should result in panic?

Reply:
It should panic!
Try to config `skip-grant-table = true` and start a TiDB (master branch), calling "flush privileges" through the mysql client you would reproduce the problem.

Reply:
No, I mean I run this unit test, it passed. Not in a mysql client.